syntax = "proto3";
import "google/protobuf/empty.proto";
option cc_generic_services = true;

message GameSceneTest
{
}


//正常或者顶号进入场景
//断线重连进入场景,断线重连分时间
service GameSceneService
{
  rpc Test (GameSceneTest) returns (google.protobuf.Empty);
}